Source/Core/Core/PowerPC/JitCommon/JitAsmCommon.h

```diff
@@ -40,6 +40,12 @@ class CommonAsmRoutinesBase
   const u8** pairedLoadQuantized;
 
   // In: array index: GQR to use.
+  // In: ECX: Address to read from.
+  // Out: XMM0: Bottom 32-bit slot holds the read value.
+  // Trashes: all three RSCRATCH
+  const u8** singleLoadQuantized;
```

Comment:
Was just a thought/suggestion. A clean-up PR at a later point is fine by me too.